influxdb3_server/src/query_executor.rs

```diff
@@ -100,13 +100,13 @@ impl<W: WriteBuffer> QueryExecutor for QueryExecutorImpl<W> {
     async fn query(
         &self,
         database: &str,
-        q: &str,
+        query: &str,
         params: Option<StatementParams>,
         kind: QueryKind,
         span_ctx: Option<SpanContext>,
         external_span_ctx: Option<RequestLogContext>,
     ) -> Result<SendableRecordBatchStream, Self::Error> {
-        info!("query in executor {}", database);
+        debug!(%database, %query, ?params, ?kind, "QueryExecutorImpl as QueryExecutor::query");
```

Comment:
Usually I'd have the individual queries be info as I expect those to be in the log. But I'd ideally be showing that at then end along with an execution time (while making sure it gets logged if there was a failure)

Reply:
ah, if the query is already getting logged elsewhere, that's good enough for me.

Reply:
Switched it back to `info!` in https://github.com/influxdata/influxdb/pull/25102/commits/2f91c5a59f73bb57a05339d238c4f1ef8d29976e

Once CI is good I can merge.